src/main/java/baseball/domain/Numbers.java

```diff
@@ -0,0 +1,50 @@
+package baseball.domain;
+
+import baseball.common.exception.input.InputException;
+
```

Comment:
java 컨벤션에 위배 되는 것 같습니다 !

Reply:
어떤 부분이 위배되는지 알려주실 수 있을까요?? 지적 감사합니다!

Reply:
3.3.3 Ordering and spacing

Imports are ordered as follows:
1. All static imports in a single block.
2. All non-static imports in a single block.
If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.
Within each block the imported names appear in ASCII sort order. (Note: this is not the same as the import statements being in ASCII sort order, since '.' sorts before ';'.)

위의 부분입니다 !! `static import와 non-static import가 있는 경우 사이에 single blank를 둬 두 블럭을 나누고 이 외에 다른 blank는 있어선 안된다` 입니다 !